pytest_splunk_addon/splunk.py

```diff
@@ -363,6 +372,9 @@ def splunk_docker(
         "password": request.config.getoption("splunk_password"),
     }
 
+    splunk_info["forwarder_host"] = splunk_info.get("host")
```

Comment:
Unused code.

Reply:
It is used in the fixture **splunk_hec_uri**  L484